_includes/header.html

```diff
@@ -2,7 +2,13 @@
 
   <div class="wrapper">
 
-    <a class="site-title" href="{{ site.baseurl }}/">{{ site.title | escape }}</a>
+    <a class="site-title" href="{{ site.baseurl }}/">
+      {% if site.logo %}
+        <img src="/{{ site.logo }}" class="site-logo" width="150" height="auto">
```

Comment:
okay, I'll switch the attributes and set the height to match `site-title { line-height: 56px; }`